weave/integrations/smolagents/smolagents_sdk.py

```diff
@@ -107,6 +107,7 @@ def get_smolagents_patcher(
             # Patch model-related classes
             get_symbol_patcher("smolagents", "TransformersModel.__call__", base),
             get_symbol_patcher("smolagents", "HfApiModel.__call__", base),
+            get_symbol_patcher("smolagents", "InferenceClientModel.__call__", base),
```

Comment:
`HfApiModel` is deprecated in favor of `InferenceClientModel`. Leaving the former for backwards compatibility.